content/javascript/concepts/objects/terms/fromEntries/fromEntries.md

```diff
@@ -0,0 +1,61 @@
+---
+Title: '.fromEntries()'
+Description: 'Returns a new object with properties from a given iterable.'
+Subjects:
+  - 'Web Development'
+Tags:
+  - 'Methods'
+  - 'Objects'
+CatalogContent:
+  - 'introduction-to-javascript'
+  - 'paths/full-stack-engineer-career-path'
+---
+
+The **`.fromEntries()`** method returns a new object with properties from a given list of key-value pairs.
```

Comment:
You should explain how the list has two elements (within individual elements), and when converting to an object, the first element is assigned as the `key` and the second as the `value`.

Reply:
Thank you! Considering the example below, I added your suggestion there instead of with this sentence. Let me know if you think we should move it up here.